### PR TITLE
docs(readme): add release badge

### PR DESCRIPTION
**Pull request by yqlbu**
### Background

As the title suggests.

### Checklist

- [ ] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/daed

### Full changelog

- docs(readme): add release badge

### Issue reference

NA

### Preview

CleanShot 2023-07-08 at 22 37 42@2x

### CODE REVIEW

**Comment by kunish** [comment]
Can we make this badge dynamic?

**Comment by yqlbu** [comment]
> Can we make this badge dynamic?

YES, but not for now. @earrmouth is working on docs-synchronization which includes a task for updating `release-badge` on the fly (trigger whenever a new release is published).

**Comment by kunish** [comment]
> > Can we make this badge dynamic?
> 
> YES, but not for now. @earrmouth is working on docs-synchronization which includes a task for updating `release-badge` on the fly (trigger whenever a new release is published).

I'm wondering if we can come up with a better solution which will reuse the same badge image url but the url is pointing to the latest release tag.

**Comment by yqlbu** [comment]
> > > Can we make this badge dynamic?
> > 
> > 
> > YES, but not for now. @earrmouth is working on docs-synchronization which includes a task for updating `release-badge` on the fly (trigger whenever a new release is published).
> 
> I'm wondering if we can come up with a better solution which will reuse the same badge image url but the badge image is showing the latest release tag.

Good point, let's do some research on this. cc @earrmouth

**Comment by kunish** [comment]
@yqlbu OK, thank you

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 <p align="center">
   <img src="https://custom-icon-badges.herokuapp.com/github/license/daeuniverse/daed?logo=law&color=orange" alt="license" />
   <img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fdaeuniverse%2Fdaed&count_bg=%235C3DC8&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false" alt="hits" />
+  <img src="https://custom-icon-badges.herokuapp.com/badge/version-v0.1.0-blue.svg?logo=rocket&logoColor=white" alt="version">
```
Comment (kecrily) [inline README.md:12]:
```suggestion
  <img src="https://img.shields.io/github/v/release/daeuniverse/daed" alt="version">
```

Dynamic, but miss a pretty icon

<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>

  <tr>
    <td>version</td>
    <td>  version
</td>
  </tr>
</table>

**Comment by yqlbu** [comment]
## Preview

CleanShot 2023-07-08 at 23 04 54@2x

**Review by kunish — APPROVED** [review]
Great, thanks guys

**Pull request merged**